diff -ur usermode-1.9.orig/Makefile usermode-1.9/Makefile
--- usermode-1.9.orig/Makefile	Wed Apr  7 21:01:01 1999
+++ usermode-1.9/Makefile	Tue Jul 13 21:32:06 1999
@@ -36,7 +36,7 @@
 install:	$(PROGS)
 	mkdir -p $(PREFIX)/usr/bin $(PREFIX)/usr/sbin
 	mkdir -p $(PREFIX)/etc/X11/applnk/System
-	mkdir -p $(PREFIX)/usr/man/man1 $(PREFIX)/usr/man/man8
+	mkdir -p $(PREFIX)/usr/share/man/man1 $(PREFIX)/usr/share/man/man8
 	$(INSTALL) -m 755 -s userinfo $(PREFIX)/usr/bin
 	$(INSTALL) -m 644 userinfo.desktop $(PREFIX)/etc/X11/applnk/System/
 	$(INSTALL) -m 755 -s usermount $(PREFIX)/usr/bin
@@ -47,11 +47,11 @@
 	$(INSTALL) -m 4755 -s userhelper $(PREFIX)/usr/sbin
 
 install-man: 	$(MANS)
-	$(INSTALL) -m 644 userinfo.1 $(PREFIX)/usr/man/man1
-	$(INSTALL) -m 644 usermount.1 $(PREFIX)/usr/man/man1
-	$(INSTALL) -m 644 userhelper.8 $(PREFIX)/usr/man/man8
-	$(INSTALL) -m 644 consolehelper.8 $(PREFIX)/usr/man/man8
-	$(INSTALL) -m 644 userpasswd.1 $(PREFIX)/usr/man/man1
+	$(INSTALL) -m 644 userinfo.1 $(PREFIX)/usr/share/man/man1
+	$(INSTALL) -m 644 usermount.1 $(PREFIX)/usr/share/man/man1
+	$(INSTALL) -m 644 userhelper.8 $(PREFIX)/usr/share/man/man8
+	$(INSTALL) -m 644 consolehelper.8 $(PREFIX)/usr/share/man/man8
+	$(INSTALL) -m 644 userpasswd.1 $(PREFIX)/usr/share/man/man1
 
 clean:	
 	rm -f *~ *.o $(PROGS)
